### PR TITLE
Text block highlight variant

### LINKED ISSUE #140: Add a variant to text block to support highlight text area
image

https://www.sunstar-foundation.org/en/awards/world-hygienist/how-to-apply

### PR DESCRIPTION
**Pull request by JiangLong2019**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #140 

Test URLs:
- Original: https://www.sunstar-foundation.org/en/awards/world-hygienist/how-to-apply
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/en/awards/world-hygienist/how-to-apply
- After: https://text-highlight--sunstar-foundation--hlxsites.hlx.live/en/awards/world-hygienist/how-to-apply

- [x] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
Text(highlight)   |

### CODE REVIEW

**Comment by JiangLong2019** [comment]
@jindaliiita I moved the logic from quote block to text block, please check

File: blocks/text/text.css [hunk 1]
```diff
@@ -36,24 +36,20 @@ main .block.text.black p {
...
-main .block.text.highlight a.button.primary {
-  color: var(--secondary);
-  background: none;
-  font-weight: var(--font-weight-medium);
-  font-size: var(--body-font-size-m);
-  line-height: 1.5em;
-  text-decoration: none;
-  padding: 0;
-  height: auto;
+.block.quote.highlight .text-content {
```
Comment (jindaliiita) [inline blocks/text/text.css:51]: Why this change ?

File: blocks/text/text.css [hunk 2]
```diff
@@ -36,24 +36,20 @@ main .block.text.black p {
...
+  padding: 1.875rem 2.5rem;
+  background: var(--transparent-blue-light-color);
+  border-left: 5px solid var(--health-beauty-orange);
 }
 
-
-main .block.text.highlight h2 {
-  margin-bottom: .5rem;
-  text-align: center;
+main .block.text.highlight .text-title {
```
Comment (jindaliiita) [inline blocks/text/text.css:46]: I am wondering if we can ask Dejan about the title.
Because i don't think its important to have it in orange color.
If he agrees on this, then we can also remove the JS code.

File: blocks/text/text.css [hunk 3]
```diff
@@ -36,24 +36,20 @@ main .block.text.black p {
 }
 
 main .block.text.highlight {
-  margin-top: 5rem;
+  margin-top: 1.5rem;
+  margin-bottom: 5rem;
+  padding: 1.875rem 2.5rem;
+  background: var(--transparent-blue-light-color);
```
Comment (jindaliiita) [inline blocks/text/text.css:42]: Not needed, you can use light-background variant

File: blocks/text/text.css (hunk 2 above)
Comment (JiangLong2019) [reply blocks/text/text.css:46]: As we talked, will keep it

File: blocks/text/text.css (hunk 1 above)
Comment (jindaliiita) [reply blocks/text/text.css]: I am talking about, i think this is not needed
`main .block.quote.highlight .text-content`

Comment (JiangLong2019) [reply blocks/text/text.css]: @jindaliiita I just keep the same with the original site. In EDS the default one is `#3c4350`, there is no big difference between the two in eyes, I removed it.

Original site:
image

**Review by jindaliiita — APPROVED** [review]
